control/plugin/client/grpc.go

```diff
@@ -132,7 +135,13 @@ func NewCollectorGrpcClient(address string, timeout time.Duration, security GRPC
 
 // NewStreamCollectorGrpcClient returns a stream collector gRPC client
 func NewStreamCollectorGrpcClient(address string, timeout time.Duration, security GRPCSecurity) (PluginStreamCollectorClient, error) {
-	p, err := newPluginGrpcClient(address, timeout, security, plugin.StreamCollectorPluginType)
+	header := metadata.New(map[string]string{
+		"MaxCollectDuration": "5s",
```

Comment:
Why are the streaming control parameters given here? I couldn't easily find any place where these items were received (I've checked in snap and snap-plugin-lib-go).